setup.cfg

```diff
@@ -16,6 +16,6 @@ line_length = 88
 multi_line_output = 3
 
 [tool:pytest]
-addopts = --cov=httpx --cov=tests --cov-report=term-missing -rxXs
+addopts = --cov=httpx --cov=tests -rxXs
```

Comment:
Should we include `--branch` as well?

Reply:
I am in favor of this.

Reply:
I’m personally a pretty hard no on branch coverage.

Keeping things simple and pinning at 100% line coverage seems to absolutely be the sweet spot for code quality, and is a clear baseline that we can aim to enforce across all encode projects.

Reply:
Personally I think if we are aiming for 100% we'd need branch coverage and rely on explicitly ignoring them in cases where it's not possible.

That being said, I'm not going to die on this particular hill 😅